src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -112,7 +114,9 @@
     private static boolean enableMilkFluid = false;
     public static final RegistryObject<Fluid> MILK = RegistryObject.of(new ResourceLocation("milk"), ForgeRegistries.FLUIDS);
     public static final RegistryObject<Fluid> FLOWING_MILK = RegistryObject.of(new ResourceLocation("flowing_milk"), ForgeRegistries.FLUIDS);
-
+    
+    public static final IRecipeType<GrindingRecipe> GRINDING = new IRecipeType<GrindingRecipe>() {};
```

Comment:
Since Forge is against static initialization, I would probably leave this null to be set during that particular phase, but this would need to be confirmed by someone on the triage or forge team since most of them usually just static init recipe types.

Reply:
Wait a minute this shouldn't be `new IRecipeType<>` in any case, it should be registering it (`IRecipeType#register`). (Notably, the registry in question is barely used, but still, there's a register method *right there*.